backend/cmd/headlamp.go

```diff
@@ -69,6 +69,8 @@ const ContextCacheTTL = 5 * time.Minute // minutes
 
 const ContextUpdateChacheTTL = 20 * time.Second // seconds
 
+const JWTExpirationTTL = 10 * time.Second //seconds
```

Comment:
The linter (`make backend-lint`) is asking for a space here: `// seconds`

Reply:
Sorry, 

I missed that, should be fixed now :)